src/client/stylesheets/style.less

```diff
@@ -549,3 +550,10 @@ hr.wide {
 	margin-right: 0.5em !important;;
 	vertical-align: text-top;
 }
+
+@media (max-width: 498px){
+	.buttons-container > button:nth-child(2) {
```

Comment:
This `buttons-container` class is not used anywhere, so this code block is unused, and can be removed.

As mentioned in the comment above, we can add a bit of space between the buttons using existing bootstrap classes instead.